scipy/stats/tests/test_qmc.py

```diff
@@ -923,6 +881,18 @@ def test_NormalQMCSeeded(self):
                                      [-0.997875, 0.399134, 1.032234]])
         assert_allclose(samples, samples_expected, atol=1e-4)
 
+        # same test with another engine
+        seed = np.random.default_rng(274600237797326520096085022671371676017)
+        base_engine = qmc.Sobol(4, scramble=True, seed=seed)
+        engine = qmc.MultivariateNormalQMC(
```

Comment:
At some point, variable names should be changed to distributions are not referred to as engines.

Reply:
Good point.